src/libs/routes.js

```diff
@@ -3,10 +3,10 @@ import { compile, pathToRegexp } from 'path-to-regexp';
 import * as Analysis from 'src/analysis/Analyses';
 import * as AnalysisLauncher from 'src/analysis/AnalysisLauncher';
 import * as AppLauncher from 'src/analysis/AppLauncher';
-import * as Environments from 'src/analysis/Environments';
 import { routeHandlersStore } from 'src/libs/state';
 import * as AzurePreview from 'src/pages/AzurePreview';
 import * as Projects from 'src/pages/billing/List/List';
+import * as Environments from 'src/pages/EnvironmentsPage';
```

Comment:
Nit: It would be nice to match the variable name to the module (EnvironmentsPage instead of Environments).